packages/chatbot-eval-mongodb-public/src/pipelines/conversationQualityCheck.ts

```diff
@@ -23,5 +23,11 @@ runPipeline({
       genRunId
     );
     await report("conversationRetrievalScoreAvg", retrievalEvalRunId);
+
+    const { _id: relevancyEvalRunId } = await evaluate(
+      "conversationAnswerRelevancy",
+      genRunId
+    );
```

Comment:
Would be nice if we could parallelize these evals in the future

Reply:
made this one earlier today! [EAI-309](https://jira.mongodb.org/browse/EAI-309) 

think the API will look like:

```ts
// eval.config.ts

export default async () => {

  return {
    // other fields 
    evaluate: {
      someEval: {
        evaluator: someEvaluatorFunc,
        concurrency: 5 // NEW!
      }
    }
  }
}

```